Sources/Workspace/InitPackage.swift

```diff
@@ -144,8 +155,26 @@ public final class InitPackage {
                     name: "\(pkgname)"
                 """)
 
+            var platforms = options.platforms
+
+            // Macros require macOS 10.15, iOS 13, etc.
+            if packageType == .macro {
+                func addIfMissing(_ newPlatform: SupportedPlatform) {
+                  if platforms.contains(where: { platform in
+                      platform.platform == newPlatform.platform
+                  }) {
+                      return
+                  }
+
+                  platforms.append(newPlatform)
+                }
+
+              addIfMissing(.init(platform: .macOS, version: .init("10.15")))
+              addIfMissing(.init(platform: .iOS, version: .init("13")))
```

Comment:
We'll need tvOS, watchOS and Driver Kit as well here, right? Maybe Catalyst, but I'd assume the automatic derivation from iOS would be enough for that.